DQM/TrackingMonitor/interface/TrackAnalyzer.h

```diff
@@ -136,6 +136,21 @@ class TrackAnalyzer
 	    , NumberOfValidRecHitsPerTrackVsEta(NULL)
 	    , NumberOfValidRecHitVsPhiVsEtaPerTrack(NULL)
 
+            , NumberOfLostRecHitsPerTrackVsPhi(NULL)
```

Comment:
Could we drop all of these and just set to `nullptr` at the declaration place? Otherwise you end up managing two different lists which are out-of-sync (e.g., `xPointOfClosestApproachToPV` isn't set to `nullptr`. This also isn't consistent they way it's done now. I would prefer a commit making a proper cleanup of this.